src/snap/grpc_export.cc

```diff
@@ -61,13 +96,37 @@ Plugin::GRPCExportImpl* Plugin::GRPCExporter::implement() {
 future<void> Plugin::GRPCExportImpl::DoExport(shared_ptr<PluginInterface> plugin, const Meta *meta) {
   this->plugin = std::move(plugin);
   this->meta = meta;
+  this->creds = configureCreds();
```

Comment:
Prefer method name consistency. The other methods use 'Credentials' this one uses 'Creds'.